CHANGELOG.md

```diff
@@ -1,9 +1,19 @@
 # Change Log
 
+## [1.3.0] - 2023-30-01
+
+Added optional Ruby and bundler version
+
+### Added
+
+- Optional Ruby version: using the `ruby-version` property you can specify Ruby version you wish to use. If missing latest Ruby version available will be used.
+- Optional bundler version: using the `bundler-version` property you can specify bundler version you wish to use. If missing latest bundler version will be used.
```

Comment:
@EmilianoMorghen this new version also add the `fastlane-env` parameter, please add a paragraph about it

Reply:
done